examples/ticks/centered_ticklabels.py

```diff
@@ -38,11 +38,12 @@
 ax.xaxis.set_major_formatter(ticker.NullFormatter())
 ax.xaxis.set_minor_formatter(dates.DateFormatter('%b'))
 
-for tick in ax.xaxis.get_minor_ticks():
-    tick.tick1line.set_markersize(0)
-    tick.tick2line.set_markersize(0)
-    tick.label1.set_horizontalalignment('center')
+# Remove the tick lines
+ax.tick_params(axis='x', which='minor', tick1On=False, tick2On=False)
 
+# Align the minor tick label
+for label in ax.get_xticklabels(minor=True):
```

Comment:
This is maybe not the best thing to loop over. But this is the method mentioned at the top, so from that perspective it makes more sense.